src/main/java/com/harrison/messaging/Consumer.java

```diff
@@ -0,0 +1,14 @@
+package com.harrison.messaging;
+
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Consumer {
+    
+    @RabbitListener(queues = "hello")
+    public void receive(String in) {
+        System.out.println(" [x] Received '" + in + "'");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS106)